Scheduler/feature_scheduler/maintel/fbs_config_sit_survey_block_t248.py

```diff
@@ -0,0 +1,200 @@
+        Ignore observations with this string in the `scheduler_note`.
+        Will ignore observations which match subsets of the string, as well as
+        the entire string. Ignoring 'mysurvey23' will also ignore 'mysurvey2'.
+    survey_name : `str` or None.
+        The name to give this survey, for debugging and visualization purposes.
+        Also propagated to the 'target_name' in the observation.
+        The default None will construct a name based on the
+        RA/Dec of the field.
+    scheduler_note : `str` or None
+        The value to include in the scheduler note.
+        The scheduler note is for internal, scheduler, use for the purposes of
+        identifying observations to ignore or include for a survey or feature.
+    readtime : `float`
+        Readout time for computing approximate time of observing
+        the sequence. (seconds)
+    filter_change_time : `float`
+        Filter change time, on average. Used for computing approximate
+        time for the observing sequence. (seconds)
+    nside : `float` or None
+        Nside for computing survey basis functions and maps.
+        The default of None will use rubin_scheduler.utils.set_default_nside().
+    flush_pad : `float`
+        How long to hold observations in the queue after they
+        were expected to be completed (minutes).
+    """
+    # Define the extra parameters that are used in the greedy survey. I
+    # think these are fairly set, so no need to promote to utility func kwargs
+
+    survey_detailers = [
+        detailers.TrackingInfoDetailer(science_program=name),
+        detailers.CameraRotDetailer(
+            min_rot=np.min(camera_rot_limits), max_rot=np.max(camera_rot_limits)
+        ),
+    ]
+
+    for filtername in sequence:
+        bfs = [
+            (
+                bf.FootprintBasisFunction(
+                    filtername=filtername,
+                    footprint=footprints,
+                    out_of_bounds_val=np.nan,
+                    nside=nside,
+                ),
+                footprint_weight,
+            ),
+            (
+                bf.SlewtimeBasisFunction(filtername=filtername, nside=nside),
+                slewtime_weight,
+            ),
+            (bf.StrictFilterBasisFunction(filtername=filtername), stayfilter_weight),
+            (
+                bf.AltAzShadowMaskBasisFunction(
+                    nside=nside,
+                    shadow_minutes=shadow_minutes,
+                    max_alt=max_alt,
+                    min_alt=30.0,
+                ),
+                0,
+            ),
+            (bf.FilterLoadedBasisFunction(filternames=filtername), 0),
+        ]
+
+        exptimes = {filter: exptime for filter in sequence}
+        nexps = {filter: nexp for filter in sequence}
+        nvisits = {filter: visits for filter in sequence}
+        weights = [val[1] for val in bfs]
+        basis_functions = [val[0] for val in bfs]
+        survey = FieldSurvey(
```

Comment:
Seems like missing required positional arguments for RA and dec?

Reply:
Thanks - fixed with new push. Note RA/dec are just chosen to be a field that should be up all night. The actual pointing will be decided by the current pointing of the telescope, plus dithers defined in the json files.

Reply:
make sure you pass a coordinate that is always visible though, or it will not be scheduled

Reply:
For my own curiosity -- If you dropped the AltAzBasisFunction and SlewTimeBasisFunction, and picked a field that was not visible, would it still not be scheduled?

Reply:
it would not...